src/sentry/mediators/sentry_apps/destroyer.py

```diff
@@ -1,17 +1,23 @@
 from __future__ import absolute_import
 
 from sentry.mediators import Mediator, Param
+from sentry.mediators.sentry_app_installations import Destroyer as InstallDestroyer
 
 
 class Destroyer(Mediator):
     sentry_app = Param('sentry.models.SentryApp')
 
     def call(self):
+        self._destroy_sentry_app_installations()
         self._destroy_api_application()
         self._destroy_proxy_user()
         self._destroy_sentry_app()
         return self.sentry_app
 
+    def _destroy_sentry_app_installations(self):
+        for install in self.sentry_app.installations.all():
+            InstallDestroyer.run(install=install)
```

Comment:
Not a necessary change, just a suggestion for the future:

I've started importing the module itself instead of renaming the classes. So like

```python
from sentry.mediators import sentry_app_installations`
...
sentry_app_installations.Destroyer.run(install=install)
```

I know it's basically the same thing, but having fewer names for objects/classes seems better to me. Otherwise we might run into a case where here we call it `InstallDestroyer` and somewhere else we call it `Destroyer`, etc. Path of least surprise.

Reply:
Also, we should add a test to the Destroyer spec to ensure it deletes all installations.